packages/skill-lesson/team/buy-more-seats.tsx

```diff
@@ -41,10 +26,12 @@ const BuyMoreSeats = ({
       quantity: debouncedQuantity,
     })
 
-  const formActionPath = buildFormActionPath({
+  const formActionPath = buildStripeCheckoutPath({
     userId,
     quantity: debouncedQuantity,
     productId,
+    bulk: true,
+    couponId: formattedPrice?.appliedMerchantCoupon?.id,
```

Comment:
These are the values that were missing, that caused the coupon to not get propagated to Stripe